openedx/core/djangoapps/course_groups/urls.py

```diff
@@ -0,0 +1,53 @@
+"""
+Cohort API URLs
+"""
+
+from django.conf import settings
+from django.conf.urls import url
+
+from . import api
+
+urlpatterns = [
+    url(
+        r'^v1/{}/settings$'.format(
```

Comment:
I think this should be renamed to `v1/settings/{}`

Reply:
:+1: